app/controllers/messages_controller.rb

```diff
@@ -1,57 +1,42 @@
 class MessagesController < ApplicationController
   def create
+    @workspace = Workspace.find(params[:workspace_id])
+
     message_creator = Messages::Creator.new(
       permitted_params,
-      channel_destination,
+      destination,
       current_user
     )
     message = message_creator.commit
-    return suceed_messaged(message) if message
 
-    failed_messaged(message_creator)
+    message ? respond_with_success(message) : respond_with_failure(message_creator)
+
+    respond_to { |format| format.turbo_stream }
```

Comment:
I might be wrong but I think this is not necessary. What I understand of `respond_to` is that you can use it to respond in multiple formats, but here you are always responding in turbo_stream format so there is no need to use it

Reply:
yeah, let's keep it in mind, it's because I didn't find a way to use the view creat.turbo_stream.erb without the respond_to